include/cscore_oo.h

```diff
@@ -395,7 +395,7 @@ class CvSource : public VideoSource {
   /// @param description Description
   void SetDescription(llvm::StringRef description);
 
-  /// Create a property.
+  /// Create a property (deprecated)
```

Comment:
If we're going to mark this deprecated, we should do the same for the Java side.  I'd prefer to not deprecate it though.